va/validation-authority.go

```diff
@@ -388,7 +388,7 @@ func (va *ValidationAuthorityImpl) validateTLSWithZName(identifier core.AcmeIden
 
 	challenge.Error = &probs.ProblemDetails{
 		Type: probs.UnauthorizedProblem,
-		Detail: fmt.Sprintf("Correct zName not found for TLS SNI challenge. Found %s",
+		Detail: fmt.Sprintf("Correct zName not found for TLS SNI challenge. Found %v",
```

Comment:
`%v` will not print any characters if `strings.Join(certs[0].DNSNames, ", ") == ""`, wheres `%#v` will print quotation marks around the string so we'd get `""`.